src/main/java/chess/piece/Pawn.java

```diff
@@ -61,20 +59,20 @@ private List<Movement> updateBlackPawnMovableDirection(Position start) {
 
     @Override
     public void checkTargetPosition(Piece targetPiece, Position start, Position target) {
-        if (targetPiece.getTeam().equals(this.getTeam())) {
+        if (this.checkSameTeam(targetPiece.getTeam())) {
```

Comment:
team을 꼭 get을 해야할까요? 다른 Piece를 받아서 아래와 같이 확인 할 수도 있을 것 같아요
getter 사용을 지양하는 연습을 습관적으로 해주세요~ 
```suggestion
        if (this.isSameTeam(targetPiece)) {
```

Reply:
getter를 삭제하고 해당 부분들은 `isSameTeam`으로 대체하도록 수정했습니다!